spec/serializers/comment_serializer_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe CommentSerializer do
+  let(:debate) { create(:debate) }
+  let(:auth_token) { debate.auth_tokens.create }
+  let(:mobile_user) { create(:edge_mobile_user, auth_token: auth_token, first_name: 'John', last_name: 'Doe') }
+  let(:comment) { create(:comment, user: mobile_user) }
+
+  it 'receives valid data' do
+    call = described_class.new(comment).to_h
```

Comment:
Można użyć `subject`.

`subject { described_class.new(comment).to_h }`

`it '...' do`
..`expect(subject).to have_key(:xxx)`
..`expect(subject[:user_initials]).to eq('J D')`
`end`